diff --git a/src/frontend/stmclient.cc b/src/frontend/stmclient.cc
index 8523f9c..8cfbdae 100644
--- a/src/frontend/stmclient.cc
+++ b/src/frontend/stmclient.cc
@@ -373,9 +373,6 @@ bool STMClient::process_user_input( int fd )
 
       lf_entered = ( (the_byte == 0x0A) || (the_byte == 0x0D) ); /* LineFeed, Ctrl-J, '\n' or CarriageReturn, Ctrl-M, '\r' */
 
-      if ( the_byte == 0x0C ) { /* Ctrl-L */
-	repaint_requested = true;
-      }
 
       network->get_current_state().push_back( Parser::UserByte( the_byte ) );		
     }
